mp/src/game/client/hud_crosshair.cpp

```diff
@@ -43,18 +86,18 @@ DECLARE_HUDELEMENT( CHudCrosshair );
 #endif
 
 CHudCrosshair::CHudCrosshair( const char *pElementName ) :
-		CHudElement( pElementName ), BaseClass( NULL, "HudCrosshair" )
+        CHudElement( pElementName ), BaseClass( NULL, "HudCrosshair" )
 {
-	vgui::Panel *pParent = g_pClientMode->GetViewport();
-	SetParent( pParent );
+    vgui::Panel *pParent = g_pClientMode->GetViewport();
+    SetParent( pParent );
```

Comment:
You can pass `g_pClientMode->GetViewport()` into the `BaseClass`  constructor and remove these two lines.